src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,412 @@
+            </td>
+            <td>
+                Exported logs include any extracted values from ingest parsing
+            </td>
+            <td>
+                Exported logs **don't** include any extracted values from query time parsing
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Number of attributes
+            </td>
+            <td>
+                A maximum of [255 attributes](/docs/logs/log-api/log-event-data/#events) is available at ingest (the actual number attributes you can parse at ingest depends on the nature of your logs)
+            </td>
+            <td>
+                You can parse a maximum of 32 temporary attributes across all rules for query time parsing
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## How to create a query time parsing rule [#how-to] 
+
+Here's an overview of the tasks for creating query time parsing rules:
+
+<Steps>
+    <Step>
+        #### Select attribute value to parse [#select-attribute]
+        You can start creating a query time parsing rule by selecting an attribute to parse:
+
+            1. In the log table or in the **Formatted** tab of the log details view, highlight an attribute value you'd like to parse, known as the anchor string. Keep the following in mind:
+                * Your highlighted text should include the value(s) you want to extract and the surrounding string characters that will help identify the location of the extracted value(s). 
+                * If your initial anchor string is anywhere in the middle of the original attribute value, include at least one character before and after the values you want to extract.
+                * If you're highlighting the entire attribute value, you don't need to worry about characters before and after the values you're extracting.
+        <Callout variant="tip">
+            You cannot highlight blob values to parse.
+        </Callout>
+            2. Click the **Create query time parsing rule** option. 
+
+                **Log table:**
+                <img
+                    title="Screenshot showing how you can begin with the logs table UI"
+                    alt="Screenshot showing how you can begin with the logs table UI"
+                    src={logsLogsTableEntryPoint}
+                />
+
+                **Log details view**:
+                <img
+                    title="Screenshot showing how you can start creating rules in the logs detail view"
+                    alt="Screenshot showing how you can start creating rules in the logs detail view."
+                    src={logsLogsDetailEntryPoint}
+                />
+    </Step>
+    <Step>
+        #### Highlight and extract values [#highlight-extractl]
+
+        After you've clicked **Create query time parsing rule**, the editor displays the string you selected for parsing:
+
+        <img
+        title="Screenshot showing the opening screen of the query editor"
```

Comment:
I see how you have the examples displayed in the screenshot but I still wonder if it would be more effective/helpful to have examples brought into the text